docs/Serverless.md

```diff
@@ -18,36 +18,31 @@ we do not test for such integration scenarios.
 The sample provided allows you to easily build serverless web applications/services
 and RESTful APIs using Fastify on top of AWS Lambda and Amazon API Gateway.
 
-*Note: This is just one possible way.*
+*Note: Using [aws-lambda-fastify](https://github.com/adrai/aws-lambda-fastify) is just one possible way.*
 
 ### app.js
 
 ```js
 const fastify = require('fastify');
 
-function init(serverFactory) {
-  const app = fastify({ serverFactory });
-  app.get('/', (request, reply) => reply.send({ hello: 'world' }));
-  return app;
-}
```

Comment:
I would rather keep the `init` function it simplifies writing unit tests for the application.

Reply:
really?
I'm testing with the inject function too... or can also call app.ready and pass app.server to supertest for example

Reply:
example without inject:

```js
const request = require('supertest')
await app.ready()
await request(app.server)
    .get('/')
    .send()
    .expect(200)
```

Reply:
It does if you want to clean up your application/state between each test run. This is important to ensure you have a clean state for each run because of internal caching, connection management etc. See https://github.com/fastify/fastify-cli/blob/master/templates/app/test/services/root.test.js as an example on how this is  done.

Reply:
If you want we can add the init function option below and mention that this could be used to simplify writing unit tests?

Reply:
or describe here the option to test with serverFactory? https://github.com/fastify/fastify/blob/master/docs/Testing.md#inject

Reply:
I'm trying to avoid spreading the bad practice of exporting singletons with state (sockets, connections, etc). This was popularized long ago, and I've never seen a project not struggling because of it.
It's not about `serverFactory`  at all, but rather a good node.js practice.

Reply:
ok, done